newIDE/app/src/ProjectManager/PublicGameProperties.js

```diff
@@ -56,6 +69,29 @@ function PublicGameProperties({
           </Trans>
         }
       />
+      {// This view is used for public game properties as well as project properties.
+      // The following properties are not shown in project properties.
+      setPlayableWithKeyboard &&
+        setPlayableWithGamepad &&
+        setPlayableWithMobile && (
+          <React.Fragment>
+            <Checkbox
```

Comment:
This is not clean, but we will probably display the check boxes in both views at some point.

Reply:
It's a good remark, I thought it'd be a good thing to only have 1 component because the properties made sense to be in both places.
But we may move towards not sharing them in the future and have different components alltogether so we can control the order the things appear